### PR TITLE
org & workspace switcher blog

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_posts/2024-05-14-organization-workspace-switcher.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+date: 2024-05-14
+author_contributor: Akshay Sharma
+categories: 
+  - Meshery
+  - Cloud
+permalink: /blog/organization-workspace-switcher
+featured-image: /assets/images/posts/2024-05-14-organization-workspace-switcher/organization-workspace-switcher.png
+---
+
+Meshery facilitates organization and workspace management, enabling diverse architectural spaces to thrive. Within the Layer5 Cloud ecosystem, organizations form the bedrock of multi-tenancy, while Meshery Workspaces provide virtual arenas for collaborative team endeavors. Delve deeper into the intricacies of organization and workspace dynamics <a href="https://docs.layer5.io/cloud/spaces" target="_blank">here</a>.
```
Comment (leecalcote) [inline collections/_posts/2024-05-14-organization-workspace-switcher.md:13]: Please don't hyperlink the word "here". Instead, hyperlink the text that you're referring to.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: collections/_posts/2024-05-14-organization-workspace-switcher.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - **Resource Sharing**: Spaces allow for seamless resource sharing among team members, fostering collaboration.
+  - **Logical Grouping**: Within Spaces, you can group related components such as environments and infrastructure designs.
+  - **Flexibility**: Spaces support various resources like Kubernetes, Prometheus, Jaeger, Nginx, and more.
+  - **Simplified Management**: Managing and deploying resources is made easy within Spaces.
+  - **Access Control**: Spaces allow you to control access to resources by granting permissions to users and teams.
+
+We are now exposing the ability to switch between different organization and different workspace,this will enable users to :
+
+- Different spaces have different level of restriction 
+- Switch to an organization where they have the required permissions.
```
Comment (leecalcote) [inline collections/_posts/2024-05-14-organization-workspace-switcher.md:30]:
```suggestion
- Switch to an organization where they have the required permissions.

Try Meshery's new organization and workspace switcher in the [Meshery Playground](https://play.meshery.io) and [let us know what you think](https://meshery.io/subscribe)!

```

@captain-Akshay "let us know what you think" could be a hyperlink to your blog post on the new Feedback component.

File: collections/_posts/2024-05-14-organization-workspace-switcher.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+categories: 
+  - Meshery
+  - Cloud
+permalink: /blog/organization-workspace-switcher
+featured-image: /assets/images/posts/2024-05-14-organization-workspace-switcher/organization-workspace-switcher.png
+---
+
+Meshery facilitates organization and workspace management, enabling diverse architectural spaces to thrive. Within the Layer5 Cloud ecosystem, organizations form the bedrock of multi-tenancy, while Meshery Workspaces provide virtual arenas for collaborative team endeavors. Delve deeper into the intricacies of organization and workspace dynamics <a href="https://docs.layer5.io/cloud/spaces" target="_blank">here</a>.
+
+## Spaces
```
Comment (leecalcote) [inline collections/_posts/2024-05-14-organization-workspace-switcher.md:15]: Change to Workspaces, please - in the heading and sentences below.

File: collections/_posts/2024-05-14-organization-workspace-switcher.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Organization
+In Meshery, organizations form the core of tenancy, grouping users effectively. These organizations host multiple teams, each accommodating various users. Users can belong to multiple teams and organizations. Teams play a key role in managing access to workspaces and their resources, enhancing efficiency and security.
+
+<a href="https://artifacthub.io" target="_blank"><img alt="Artifact Hub Kinds" src="/assets/images/posts/2024-05-14-organization-workspace-switcher/organization_units.svg" width="50%" align="right" /></a>
+
+## Workspaces
+Workspaces empower collaboration with your teams, enabling resource control and activity tracking. Organizations support multiple teams, with each team accommodating diverse users. Users have the flexibility to join multiple teams and organizations, fostering fluidity and efficiency in workflow management.
```
Comment (ritiksaxena124) [inline collections/_posts/2024-05-14-organization-workspace-switcher.md:21]:
```suggestion
Workspaces empower collaboration with your teams, enabling resource control and activity tracking. Workspaces support multiple teams, with each team accommodating diverse users. Users have the flexibility to join multiple teams and organizations, fostering fluidity and efficiency in workflow management.
```

**Review by ritiksaxena124 — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@captain-Akshay Thank you for your contribution.
Let's discuss this on Websites's call on Monday. Add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

**Comment by sudhanshutech** [comment]
@captain-Akshay ready to publish?
Any new gifs or content needed to add?

**Comment by sudhanshutech** [comment]
@captain-Akshay is this completed?

File: assets/images/posts/2024-06-01-organization-workspace-switcher/org-workspace-meshmap.png
Comment (leecalcote) [inline assets/images/posts/2024-06-01-organization-workspace-switcher/org-workspace-meshmap.png:1]: Nice graphic!

**Comment by sudhanshutech** [comment]
@captain-Akshay is this completed?

**Comment by Deepak0320Singhal** [comment]
@captain-Akshay Thank you for your contribution.
Let's discuss this on Websites's call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit)

**Pull request closed without merging**